unraid-ui/src/components/common/loading/index.ts

```
@@ -1,5 +1,5 @@
-import Bar from "./Bar.vue";
-import Error from "./Error.vue";
-import Spinner from "./Spinner.vue";
+import Bar from '@app/components/common/loading/Bar.vue';
+import Error from '@app/components/common/loading/Error.vue';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)